actions.go

```diff
@@ -46,18 +59,51 @@ type Action struct {
 	Information string
```

Comment:
Please update the documentation in /docs. Especially the examples for adding actions

Reply:
docs will be addressed in a separate commit